spikeextractors/extractors/openephysextractors/openephysextractors.py

```diff
@@ -19,9 +19,9 @@ class OpenEphysRecordingExtractor(RecordingExtractor):
     mode = 'folder'
     installation_mesg = "To use the OpenEphys extractor, install pyopenephys: \n\n pip install pyopenephys\n\n"  # error message when not installed
 
-    def __init__(self, folder_path, *, experiment_id=0, recording_id=0, dtype='float'):
+    def __init__(self, folder_path, *, experiment_id=0, dtype='int32'):
         assert HAVE_OE, self.installation_mesg
-        assert dtype == 'int16' or 'float' in dtype, "'dtype' can be int16 (memory map) or 'float' (load into memory)"
+        assert dtype == 'int16' or 'int32' in dtype, "'dtype' can be int16 or 'int32' (both memory map)"
```

Comment:
The pyopenephys package only supports float and int16. The Open Ephys hardware records int16 (that can be memmapped). However, to get uV, one needs to multiply by the gain, which is done at the `get_traces` level. 
Please revert this change

Reply:
Change reverted